intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -38,6 +38,8 @@ LLM Runtime supports the following models:
 |[ChatGLM-6B](https://huggingface.co/THUDM/chatglm-6b), [ChatGLM2-6B](https://huggingface.co/THUDM/chatglm2-6b)| ✅ | ✅ |  Latest |
 |[Baichuan-13B-Chat](https://huggingface.co/baichuan-inc/Baichuan-13B-Chat), [Baichuan2-13B-Chat](https://huggingface.co/baichuan-inc/Baichuan2-13B-Chat)| ✅ | ✅ |  Latest |
 |[Mistral-7B](https://huggingface.co/mistralai/Mistral-7B-v0.1)| ✅ | ✅ | 4.34.0 or newer |
+|[Qwen-7B](https://huggingface.co/mistralai/Mistral-7B-v0.1), [Qwen-14b](https://huggingface.co/Qwen/Qwen-14B-Chat)| ✅ | ✅ |  Latest |
```

Comment:
7B link is  mistral ?

Reply:
https://huggingface.co/Qwen/Qwen-7B-Chat)

Reply:
sorry,my false

Reply:
fixed